src/routes/game/GameView.vue

```diff
@@ -410,7 +410,7 @@
         @points="playPoints"
         @target="beginTargeting"
       />
-      <GameDialogs @clear-selection="clearSelection" @handle-error="handleError" />
+      <GameDialogs @clear-selection="clearSelection" @handle-error="handleError($event)" />
```

Comment:
I don't think you need to pass it the event, I believe it will automatically be passed the event object

Reply:
Correct, the second argument passed to `$emit(eventName, eventData)` is passed to the corresponding event handler w/o needing to specifing $event as an argument of the handler ([docs](https://vuejs.org/guide/components/events.html#event-arguments))
```suggestion
      <GameDialogs @clear-selection="clearSelection" @handle-error="handleError" />
```